docs/agents/sematext-agent/kubernetes/configuration.md

```diff
@@ -47,4 +47,58 @@ You can adjust the configuration of Sematext Agent with additional environment v
 | JOURNAL_RETRY_INTERVAL | Specifies how often journal directory is scanned for failed events. Default interval is `30s`. You can specify a different interval in either seconds, minutes or hours (`JOURNAL_RETRY_INTERVAL=5m`)                                                                                                                                                                            |
 | AUTODISCO_TEMPLATES_PATH | Defines the location of the `autodisco.yml` file that contains definitions of patterns involved in app auto-discovery.                                                                                                                                                                                                                                                           |
 | HOSTNAME_ALIAS | When specified it overrides the original host name.                                                                                                                                                                                                                                                                                                                              |
+Your documentation is quite clear, but I've made a few adjustments for better clarity and readability. Here's the revised version:
 
+## Populating Environment Variables in Kubernetes
+
+To populate environment variables in Kubernetes, you need to propagate the variables to the pods by including them in the DaemonSet manifest within the `env` section. Consider the following example manifest:
+
+```yaml
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: sematext-agent
+  labels:
+    app: sematext-agent
+spec:
+  selector:
+    matchLabels:
+      app: sematext-agent
+ ...
+          env:
+            - name: AUTODISCO_VECTOR_SERVICE_ACCOUNT
+              value: sematext-agent-vector
+            - name: INFRA_TOKEN
+              value: <your-infra-token>
+            - name: KUBERNETES_CLUSTER_ID
+              value: <REPLACE_WITH_CLUSTER_NAME>
+            - name: API_SERVER_PORT
+              value: "8675"
+            - name: REGION
+              value: US
+          livenessProbe:
+            httpGet:
+              path: /health
+              port: 8675
+ ...
+```
+
+You can add additional environment variables by following the `name` and `value` format, where the first is the name of the variable and the second is the value. For example, to skip certain containers based on the image like `nginx`, add the following lines:
+
+```yaml
+            - name: CONTAINER_SKIP_BY_IMAGE
```

Comment:
Maybe we can provide an example with skipping 2 images.... for those who skipped reading how to specify multiple values.

Reply:
added one example in both, containers and kubernetes